react-components/src/data-providers/core-dm-provider/cogniteCadNodeSourceWithProperties.ts

```diff
@@ -0,0 +1,23 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+import { type SourceSelectorV3 } from '@cognite/sdk/dist/src';
+import { COGNITE_CAD_NODE_SOURCE } from './dataModels';
+
+export const cogniteCadNodeSourceWithProperties = [
+  {
+    source: COGNITE_CAD_NODE_SOURCE,
+    properties: [
```

Comment:
Do you need the properties from Describable like name, description, tags and aliases? If so, it would be nice to compose this property list of Describable properties and Cad Node Properties

Reply:
Good point, I'll do that